### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by santhu342**
Update php development

### CODE REVIEW

**Comment by santhu342** [comment]
Php development app update

File: README.md [hunk 1]
```diff
@@ -80,6 +80,14 @@ We'll organise into different streams here. One kind request for motivation: Ple
 * Instagram Clone
 * Netflix Clone
 
+### PHP Development
```
Comment (praveenscience) [inline README.md:83]: Similar to this?
https://github.com/praveenscience/Hacktoberfest-Project-Ideas/pull/75/files

**Review by praveenscience — CHANGES_REQUESTED** [review]
Is it same or different?

**Comment by praveenscience** [comment]
Closing this as invalid.

**Pull request closed without merging**